src/Sarif.WorkItemFiling/Extensions.cs

```diff
@@ -218,7 +45,7 @@ public static string GetProjectName(this Uri projectUri)
             return WebUtility.UrlDecode(projectName);
         }
 
-        public static string GetAccountUriString(this Uri projectUri)
+        public static string GetAccountName(this Uri projectUri)
```

Comment:
>GetAccountName [](start = 29, length = 14)

Just an idea and not for this PR: I think it might be easier to take in the name of the project and the organization for Azure DevOps, then we wouldn't need to do any URL parsing.

Reply:
That is true. However, if this information is already expressed in the URL then it is redundant to require that it be passed again. Also creates new negative cases where the alternate identifier might not match the URL. For ADO URLs, the account and project are definitely codified in a specific location, so I think this parsing is a fair approach. Thoughts?

---
In reply to: [375957323](https://github.com/microsoft/sarif-sdk/pull/1744#discussion_r375957323) [](ancestors = 375957323)

Reply:
Sorry I forgot to mention that either we require the URL approach or org&projectname approach. I wouldn't do both. If we are to take in the org name and project name, we can construct the URL based on the input params from org/project, this approach avoids us from having to parse the URL all together. Given that said, I think it's fine to keep what we have as it's working, just wanted to see if my approach might be easier to manage.

---
In reply to: [375990369](https://github.com/microsoft/sarif-sdk/pull/1744#discussion_r375990369) [](ancestors = 375990369,375957323)

Reply:
Makes sense! This might require us to have a host identifier (GH vs. ADO) or to infer this based on having different command-line arguments. Let's defer this conversation as you suggest, but definitely let's consider in a future change. 

---
In reply to: [376006810](https://github.com/microsoft/sarif-sdk/pull/1744#discussion_r376006810) [](ancestors = 376006810,375990369,375957323)